boards/arm/mimx8mm_phyboard_polis/doc/index.rst

```diff
@@ -0,0 +1,340 @@
+   - &ocram_sys
+   - &ocram_s_sys
+
+
+Starting the M4-Core via U-Boot
+===============================
+
+Load the compiled zephyr.bin to memory address 0x4800000.
+This should output something like this:
+
+.. code-block:: console
+
+   u-boot=> tftp 0x48000000 192.168.3.10:zyphr.bin
+   Using ethernet@30be0000 device
+   TFTP from server 192.168.3.10; our IP address is 192.168.3.11
+   Filename 'zepyhr.bin'.
+   Load address: 0x48000000
+   Loading: ##
+            2 KiB/s
+   done
+   Bytes transferred = 27240 (6a68 hex)
+
+Because it's not possible to load directly to the TCM memory area you have to
+copy the binaries. The last argument given is the size of the file in bytes,
+you can copy it from the output of the last command.
+
+.. code-block:: console
+
+   u-boot=> cp.b 0x48000000 0x7e0000 27240
+
+And finaly starting the M4-Core at the right memory address:
+
+.. code-block:: console
+
+   u-boot=> bootaux 0x7e0000
+   ## Starting auxiliary core stack = 0x20003A58, pc = 0x1FFE1905...
+
+
+Starting the M4-Core via remoteproc
+===================================
+
+Copy the zepyhr.elf to ``/lib/firmware`` on the target. Maybe a Zephyr sample
+will be included in a future BSP release.
+
+.. note::
+   In order to use remoteproc you have to add ``imx8mm-phycore-rpmsg.dtbo`` at
+   the end of the line in the ``/boot/bootenv.txt``, then reboot the target.
+
+.. warning::
+   Remoteproc only reads firmware files from the ``/lib/firmware`` directory!
+   If you try to load a binary from another location unexpected errors will
+   occur!
+
+To load and start a firmware use this commands:
+
+.. code-block:: console
+
+   target$ echo /lib/firmware/zepyhr.elf > /sys/class/remoteproc/remoteproc0/firmware
+   target$ echo start > /sys/class/remoteproc/remoteproc0/state
+   [   90.700611] remoteproc remoteproc0: powering up imx-rproc
+   [   90.706114] remoteproc remoteproc0: Direct firmware load for /lib/firmware/zepyhr.elf failed w2
+   [   90.716571] remoteproc remoteproc0: Falling back to sysfs fallback for: /lib/firmware/zepyhr.elf
+   [   90.739280] remoteproc remoteproc0: Booting fw image /lib/firmware/zepyhr.elf, size 599356
+   [   90.804448] remoteproc remoteproc0: remote processor imx-rproc is now up
+
+
+The M4-Core is now started up and running. You can see the output from Zephyr
+on UART4.
+
+Debugging
+=========
+
+The PhyBOARD Polis can be debugged using a JTAG Debugger.
+The easiest way to do that is to use a SEGGER JLink Debugger and Phytec's
+``PEB-EVAL-01`` Shield, which can be directly connected to the JLink.
+You can find the JLink Software package here: `JLink Software`_
+
+.. figure:: img/PEB-EVAL-01.jpg
+   :alt: PEB-EVAL-01
+   :width: 350
+
+   PEB-EVAL-01
+
+To debug efficiently you have to use multiple terminals:
+
+After connecting everything and building with west use this command while in
+the directory of the program you build earlier to start a debug server:
+
+.. code-block:: console
+
+   host$ west debugserver
```

Comment:
```west debug``` also works fine. But I prefer the way I documented. If you think it's better to have it documented with ```west debug```, I can change it.

Reply:
I'm not opposed to the documentation here, but I would add a note clarifying that `west debug` or `west debugserver` are both supported, just to avoid confusing users